extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -45,6 +45,7 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverA9IncontentBoxadCountries';
 		$vars[] = 'wgAdDriverA9OptOutCountries';
 		$vars[] = 'wgAdDriverA9VideoBidderCountries';
+		$vars[] = 'wgAdDriverAE3NativeSearchCountries';
```

Comment:
Can we make it consistent with other variables? e.g. `wgAdDriverNativeSearchDesktopCountries`